reference/promise-types/edit_line.markdown

```diff
@@ -411,7 +412,37 @@ mark out the region of a file to be edited.
      }
 ```
 
-If you want to match from a starting location to the end of the file
-(even if there are other lines matching `select_start` intervening),
-then just omit the `select_end` promise and the selected region will run
-to the end of the file.
+By default end of the file will be used as end of the region if `select_end` pattern was not 
```

Comment:
Would change to "By default, the end of the file is used as end of region if `select_end` does not match anything."